diff --git a/src/main/java/com/licel/jcardsim/remote/VSmartCard.java b/src/main/java/com/licel/jcardsim/remote/VSmartCard.java
index ded6586..a35f46f 100644
--- a/src/main/java/com/licel/jcardsim/remote/VSmartCard.java
+++ b/src/main/java/com/licel/jcardsim/remote/VSmartCard.java
@@ -22,6 +22,7 @@ import com.licel.jcardsim.base.Simulator;
 import static com.licel.jcardsim.base.Simulator.ATR_SYSTEM_PROPERTY;
 import static com.licel.jcardsim.base.Simulator.DEFAULT_ATR;
 import com.licel.jcardsim.base.SimulatorRuntime;
+import com.licel.jcardsim.base.PersistentSimulatorRuntime;
 import java.io.BufferedReader;
 import java.io.File;
 import java.io.FileInputStream;
@@ -110,7 +111,8 @@ public class VSmartCard {
         } catch (Throwable ex) {
             ex.printStackTrace(System.err);
             System.out.println("Failed to load the instance! Will use the default SimulatorRuntime");
-            simRuntime = new SimulatorRuntime();
+            //simRuntime = new SimulatorRuntime();
+            simRuntime = new PersistentSimulatorRuntime();
         }
         sim = new Simulator(simRuntime);
         
